Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.cpp

```diff
@@ -203,6 +203,11 @@ namespace AzToolsFramework
             }
         }
 
+        const AZStd::vector<AZ::Uuid>& ComponentModeCollection::GetComponentTypes() const
+        {
+            return m_activeComponentTypes;
```

Comment:
question: Might it be possible to add an assert here to make sure it's only called when in a Component Mode?